iceoryx_posh/include/iceoryx_posh/popo/request.hpp

```diff
@@ -46,7 +46,8 @@ class Request
 
   public:
     /// @brief Constructor for a Request used by the server/client
-    /// @param smartChunkUniquePtr is a `rvalue` to a `cxx::unique_ptr<T>` with to the data of the encapsulated type T
+    /// @param smartChunkUniquePtr is a `rvalue` to a `iox::unique_ptr<T>` with to the data of the encapsulated type
```

Comment:
We need to change the back-ticks to single quotes since Axivion will complain about them ... maybe something that should be done at once in the whole condebase

Reply:
True, I'll do it right here for `unique_ptr` nevertheless.